dump/dump.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package dump provides functions to dump values to stdout and strings.
```

Comment:
Perhaps you can take the first sentence from this package's readme, i.e:
Package dump can dump a Go data structure to Go source file, so that it can be statically embedded into other code.